### ISSUE TITLE
Add format specifier for tzdata timezone

### ISSUE THREAD

**Issue by iamed2** [JuliaTime/TimeZones.jl#128]
It would be nice to parse and output "2013-01-01 13:12:11.000000 America/Winnipeg", for example.

**Comment by omus** [comment]
I agree that would be nice. I mostly haven't done this already as I'm not sure what the format character should be or if there is a standard character for this. Off the top of my head I would use `t` or `T` (for `TimeZone`).
